src/components/logs/forms/glucose.tsx

```
@@ -0,0 +1,249 @@
+            <form onSubmit={(e) => {
+                e.preventDefault()
+                e.stopPropagation()
+
+                form.handleSubmit()
+            }} className="grid gap-4 mt-3">
+                <div className="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3 gap-4">
+                    <div className="grid gap-2">
+                        <Label htmlFor="glucose">Glucose</Label>
+                        <form.Field
+                            name="glucose"
+                            children={(field) => (
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)